config/locales/en/dashboard.yml

```diff
@@ -14,6 +14,7 @@ en:
         in_progress: In progress
         submitted: Submitted
         returned: Returned
+      no_records: There are no applications in progress
```

Comment:
We can now remove this locale, and the other one as well, as now it is using `shared.errors` right?